clients/src/test/java/org/apache/kafka/common/message/StructMessageTest.java

```diff
@@ -27,46 +28,49 @@
 import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-public class NullableStructMessageTest {
+public class StructMessageTest {
```

Comment:
Renamed `NullableStructMessageTest` to `StructMessageTest` since a non-nullable struct field is added.